modules/nf-core/software/bwamem2/mem/main.nf

```diff
@@ -33,14 +31,16 @@ process BWAMEM2_MEM {
     def prefix     = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
     def read_group = meta.read_group ? "-R ${meta.read_group}" : ""
     """
-    bwa-mem2 mem \
-        $options.args \
-        $read_group \
-        -t $task.cpus \
-        $fasta \
-        $reads \
-        | samtools $options.args2 --threads $task.cpus -o ${prefix}.bam -
+    INDEX=`find -L ./ -name "*.amb" | sed 's/.amb//'`
```

Comment:
oh cool, the other files are not necessary?

Reply:
As I discovered, thanks to @drpatelh